osu.Game/Beatmaps/Beatmap.cs

```diff
@@ -107,9 +107,12 @@ public double GetMostCommonBeatLength()
                                 // Aggregate durations into a set of (beatLength, duration) tuples for each beat length
                                 .GroupBy(t => Math.Round(t.beatLength * 1000) / 1000)
                                 .Select(g => (beatLength: g.Key, duration: g.Sum(t => t.duration)))
-                                // Get the most common one, or 0 as a suitable default
+                                // Get the most common one, or 0 as a suitable default (see handling below)
                                 .OrderByDescending(i => i.duration).FirstOrDefault();
 
+            if (mostCommon.beatLength == 0)
+                return TimingControlPoint.DEFAULT_BEAT_LENGTH;
```

Comment:
This fallback was accidentally removed in https://github.com/ppy/osu/pull/11495.

I'm mostly crossing my fingers that if it used to work that way and nobody has noticed thus far or reported anything, then no real map will hit this edge case...